### PR TITLE
Pass timeout as param

### LINKED ISSUE #4: Implement timeout parameter?
Currently (in my branch, hopefully soon on master) the timeout is hardcoded to 25 seconds. Do we want to pass that as a parameter too?

### PR DESCRIPTION
**Pull request by brittanydionigi**
Resolves #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -202,9 +205,7 @@ function startSimulatorProcess(options) {
 }
 
 
-function waitUntilSimulatorIsReady(port) {
-
-  var maxTimeout = 25000;
+function waitUntilSimulatorIsReady(port, maxTimeout) {
```
Comment (tofumatt) [inline index.js]: Can this be an options hash, so we don't have to remember order of arguments? That'd be nicer ^_^

Comment (brittanydionigi) [inline index.js]: Yes it can =D brb

**Comment by tofumatt** [comment]
I'll merge this but we'll need to update the example too. I'll do that now :smile:

**Comment by tofumatt** [comment]
Nevermind, we aren't using that anywhere in the examples; I'm dumb.

**Pull request merged**